### PR TITLE
SD-835: drone file changed to push latest tagged image to repo

### PR DESCRIPTION
**Pull request by SarahJaneLuff**
**Changes**
- Added imaged tagged server-latest, alerts-latest and lookup-latest to repo, to be used by modern slavery pipeline.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -27,9 +27,15 @@ steps:
     - docker tag server-$${DRONE_COMMIT_SHA} quay.io/ukhomeofficedigital/modern-slavery-data-service:server-$${DRONE_COMMIT_SHA}
     - docker tag alerts-$${DRONE_COMMIT_SHA} quay.io/ukhomeofficedigital/modern-slavery-data-service:alerts-$${DRONE_COMMIT_SHA}
     - docker tag lookup-$${DRONE_COMMIT_SHA} quay.io/ukhomeofficedigital/modern-slavery-data-service:lookup-$${DRONE_COMMIT_SHA}
+    - docker tag server-$${DRONE_COMMIT_SHA} quay.io/ukhomeofficedigital/modern-slavery-data-service:server-latest
+    - docker tag alerts-$${DRONE_COMMIT_SHA} quay.io/ukhomeofficedigital/modern-slavery-data-service:alerts-latest
+    - docker tag lookup-$${DRONE_COMMIT_SHA} quay.io/ukhomeofficedigital/modern-slavery-data-service:lookup-latest
     - docker push quay.io/ukhomeofficedigital/modern-slavery-data-service:server-$${DRONE_COMMIT_SHA}
     - docker push quay.io/ukhomeofficedigital/modern-slavery-data-service:alerts-$${DRONE_COMMIT_SHA}
     - docker push quay.io/ukhomeofficedigital/modern-slavery-data-service:lookup-$${DRONE_COMMIT_SHA}
+    - docker push quay.io/ukhomeofficedigital/modern-slavery:server-latest
```
Comment (Alex-Swann) [inline .drone.yml:36]: You don't need to do this. Lines 33-35 are the images you have built which are tagged with <service>-latest. Lines 36-38 will be pushing images that haven't been built so do not exist.

**Review by Alex-Swann — CHANGES_REQUESTED** [review]
See comment

File: .drone.yml (hunk 1 above)
Comment (Alex-Swann) [reply .drone.yml:36]: Ah I see they are pushing by tag and not image name. Therefore you need line 36-38 to be
```
docker push quay.io/ukhomeofficedigital/modern-slavery-data-service:server-latest
docker push quay.io/ukhomeofficedigital/modern-slavery-data-service:alerts-latest
docker push quay.io/ukhomeofficedigital/modern-slavery-data-service:lookup-latest
```

Comment (SarahJaneLuff) [reply .drone.yml]: cools, thanks. Changes these lines.

**Review by Alex-Swann — COMMENTED** [review]
all good

**Pull request merged**